src/js/components/Sidebar.js

```diff
@@ -71,7 +70,7 @@ var Sidebar = React.createClass({
       defaultMenuItems
     );
 
-    return _.map(menuItems, function (routeKey) {
+    return menuItems.map(function (routeKey) {
```

Comment:
`map` doesn't take a 2nd argument. this whole block of code should not be working right now (i haven't tested)

use `=>`